tests/tests_resize.yml

```diff
@@ -428,3 +450,118 @@
 
         - name: Verify role results
           include_tasks: verify-role-results.yml
+
+      # For XFS
+
+    - name: Test xfs
+      block:
+      - name: Run test on supported platforms
+        when: ((is_fedora and blivet_pkg_version is version("3.12.1-3", ">=")) or
```

Comment:
This version is not yet available, keeping the PR as draft for now.

Reply:
python-blivet-3.12.1-3 is now available in rawhide and should be available in F42 repos soon.